packages/l2b/src/commands/DownloadShapes.ts

```diff
@@ -0,0 +1,78 @@
+import { writeFileSync } from 'fs'
+import { mkdirSync } from 'fs'
+import { join } from 'path'
+import {
+  TemplateService,
+  flattenStartingFrom,
+  getChainConfig,
+  getDiscoveryPaths,
+} from '@l2beat/discovery'
+import { flatteningHash } from '@l2beat/discovery/dist/flatten/utils'
+import { getExplorerClient } from '@l2beat/discovery/dist/utils/IEtherscanClient'
+import { CliLogger, HttpClient } from '@l2beat/shared'
+import { command, positional, string } from 'cmd-ts'
+import { rimraf } from 'rimraf'
+
+export const DownloadShapes = command({
+  name: 'download-shapes',
+  description: 'Download all Solidity files for shapes defined in a template',
+  args: {
+    template: positional({
```

Comment:
I'd allow the user to pass `--all` and it would iterate all templates with shapes and download all things